tap/tcp_reader.go

```diff
@@ -48,6 +49,13 @@ func NewTcpReader(ident string, tcpId *api.TcpID, parent *tcpStream, isClient bo
 
 func (reader *tcpReader) run(options *api.TrafficFilteringOptions, wg *sync.WaitGroup) {
 	defer wg.Done()
+
+	if os.Getenv("MIZU_TAPPER_NO_DISSECTORS") == "true" {
+		b := bufio.NewReader(reader)
+		io.ReadAll(b)
+		return
+	}
```

Comment:
I would not call `run` at all. `io.ReadAll(b)` will consume memory.

Reply:
We have to empty tcpReader's message queue.
Otherwise `tcpReassemblyStream.ReassembledSG` hands on `reader.sendMsgIfNotClosed`.

Reply:
`reader.close()` closes the queue. But anyway, it depends what you really want to achieve.